be/models/user.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema({
+  username: String,
+  password: String,
+});
```

Comment:
are we  saving other fields too for database?
like:
const userSchema = new mongoose.Schema({
    first_name: String,
     last_name: String,
     age: Number,
     height: Number,
     weight: Number,
     gender: String

    
})

Reply:
const userSchema = new mongoose.Schema({
username: String,
password: String,
age: Number,
height: Number,
weight: Number,
gender: String

})

Reply:
You are right. As agreed, let's just use username and password only?